sudhir.io/understanding-connections-pools.md

```diff
@@ -77,16 +77,16 @@ A rule of thumb would be to place one worker for each CPU core you have availabl
 
-Now this raises a new problem—having workers * threads database connections is great for application server performance, but will wreak havoc on a database like PostgreSQL and sometimes on MySQL. Depending on how much RAM (in the case of Postgres) and how much CPU (in the case of MySQL) you have, this configuration might not work for you. You might decide to reduce the pool or max threads value to make sure you have fewer connections, or the workers value, or both. Depending on your application, all are likely to have the same effect—if every request requires database queries, the number of database connections you have is the ultimate bottleneck for the number of requests you can handle. But if some requests can function without database access, you can get away with keeping the workers and threads count high, and the pool number relatively low—that way many threads will service your requests, but only a smaller subset will actually fight over the database connections in the pool as and when they need them. You can also to this if you take over connection management in your code and make sure to checkout and release connections efficiently—this is especially important if you're making network requests between database calls.
+Now this raises a new problem—having workers * threads database connections is great for application server performance, but will wreak havoc on a database like PostgreSQL and sometimes on MySQL. Depending on how much RAM (in the case of Postgres) and how much CPU (in the case of MySQL) you have, this configuration might not work for you. You might decide to reduce the pool or max threads value to make sure you have fewer connections, or the workers value, or both. Depending on your application, all are likely to have the same effect—if every request requires database queries, the number of database connections you have is the ultimate bottleneck for the number of requests you can handle. But if some requests can function without database access, you can get away with keeping the workers and threads count high, and the pool number relatively low—that way many threads will service your requests, but only a smaller subset will actually fight over the database connections in the pool as and when they need them. You can also do this if you take over connection management in your code and make sure to checkout and release connections efficiently—this is especially important if you're making network requests between database calls.
```

Comment:
The colored git diff isn't very helpful here. The only typo here is that "You can also to this" should be changed to "You can also do this"